packages/shared/lib/core/wallet/utils/buildFoundryOutputData.ts

```diff
@@ -18,19 +18,19 @@ export async function buildFoundryOutputData(
     metadata: IIrc30Metadata,
     accountId: string
 ): Promise<FoundryOutputBuilderParams> {
-    const immutableAliasUnlockCondition = new ImmutableAccountAddressUnlockCondition(
+    const immutableAccountUnlockCondition = new ImmutableAccountAddressUnlockCondition(
         new AccountAddress(api.bech32ToHex(accountId))
     )
 
-    const unlockConditions: UnlockCondition[] = [immutableAliasUnlockCondition]
+    const unlockConditions: UnlockCondition[] = [immutableAccountUnlockCondition]
 
     const tokenScheme = new SimpleTokenScheme(BigInt(circulatingSupply), BigInt(0), BigInt(totalSupply))
 
     const metadataFeature = new MetadataFeature(Converter.utf8ToHex(JSON.stringify(metadata)))
 
     const immutableFeatures: Feature[] = [metadataFeature]
 
-    const serialNumber = await getSerialNumberFromAliasOutput(accountId)
+    const serialNumber = await getSerialNumberFromAccountOutput(accountId)
```

Comment:
`getSerialNumberFromAccountOutput` receives an account address. We could also take the opportunity to change the name of the function to `getSerialNumberFromAccountAddress`